templates/build.gradle

```diff
@@ -61,19 +60,6 @@ if (file(INJECT).exists()) {
     }
 }
 
```

Comment:
Should probably remove the inject folder from the repo for all versions that we can. I don't know if the `StartClient.java` will work for 1.12.2 and it's getting late for me to test it today. 😄